dualscreeninfo/src/dualscreeninfo/index.ts

```diff
@@ -3,30 +3,48 @@
  * Licensed under the MIT License.
  */
 
-import { DualScreenInfoEvent, SpannedChangeHandler, WindowRect } from '../types';
+import { DualScreenInfoEvent, DualScreenInfoPayload, SpannedChangeHandler, WindowRect } from "../types";
 import { EmitterSubscription, NativeEventEmitter, NativeModules } from "react-native";
 
+declare module 'react-native' {
+	namespace NativeModules {
+		export interface DualScreenInfo {
+			isDualScreenDevice: boolean
+			hingeWidth: number
+		}
+	}
+}
+
 export interface ExposedNativeMethods {
-    getWindowRects: () => Promise<WindowRect[]>;
-		isSpanned: () => Promise<boolean>;
+	addEventListener: (
+		type: DualScreenInfoEvent,
+		handler: SpannedChangeHandler
+	) => void;
+	removeEventListener: (
+		type: DualScreenInfoEvent,
+		handler: SpannedChangeHandler
+	) => void;
 }
 
 interface IDualScreenInfoModule extends ExposedNativeMethods {
     isDualScreenDevice: boolean;
 		hingeWidth: number;
-    addEventListener: (
-        type: DualScreenInfoEvent,
-        handler: SpannedChangeHandler
-    ) => void;
-    removeEventListener: (
-        type: DualScreenInfoEvent,
-        handler: SpannedChangeHandler
-    ) => void;
+	  isSpanning: boolean;
+		windowRects: WindowRect[];
```

Comment:
nit-pick:  mixing space/tabs.  Note that on our team we prefer spaces to tabs.  I don't feel too strongly about this as long as it's consistent.

Reply:
fixed